chapters/remote/remote-flows.md

```diff
@@ -0,0 +1,71 @@
+# Remote Flows with JinaD
+
+### Prerequisites
+
+Before you start, make sure you have read the [prerequisites for using JinaD](https://docs.jina.ai/chapters/remote/jinad.html#prerequisites).
+
+## REST API
+
+[Refer the detailed API reference for JinaD](https://api.jina.ai/daemon/#tag/flows)
+
+A common use case is running the Flow on the remote with pods distributed across different machines.
+
+#### Examples
+
+Define `flow.yml` with the correct `JinaD` host details.
+
+```yaml
+jtype: Flow
+pods:
+    - uses: _logforward
+      host: 1.2.3.4:8000
```

Comment:
mention somewhere that this is dependent on the port you set

Reply:
FIxed